src/components/Map/useMapIcons.tsx

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  icon: {
+    backgroundColor: theme.palette.TwClrBaseWhite,
+    border: `1px solid ${theme.palette.TwClrBgTertiary}`,
+    borderRadius: theme.spacing(0.5),
+    margin: theme.spacing(0, 0.5),
+    width: '20px',
+    height: '20px',
+  },
+}));
+
+export type MapIconType = 'polygon' | 'slice' | 'trash';
+
+type MapIcons = Record<MapIconType, JSX.Element>;
+
+export default function useMapIcons(): MapIcons {
```

Comment:
Is there a reason you went for a hook instead of a component?

Reply:
I need to switch this to components, I recently got some icon assets from Chudo. Will do that in a follow up PR for https://terraformation.atlassian.net/browse/SW-4697